packages/bindings/schema/marker.json

```diff
@@ -6,6 +6,7 @@
   "required": [
     "account_number",
     "address",
+    "allow_forced_transfer",
```

Comment:
It's probably okay to not require this new field. Since it's a boolean, not having the field would be the same as `"allow_forced_transfer": false`. I don't know what all depends on this, but by not requiring it, there's a better chance that it doesn't break that stuff.

There's a good chance I don't understand what this `"required":` list is though, and am thusly speaking gibberish.

Same goes for `packages/bindings/schema/provenance_msg.json`.